### DISCUSSION TITLE
Estimated date for next release?

### DISCUSSION THREAD

**Discussion by MauritsMonteyne** [celery/kombu#2032] [Q&A] [answered]
Hi

I noticed a [fix](https://github.com/celery/kombu/commit/dcb43be8916865e066643a3b8fd1bd6fb767601a) was recently merged for an issue that caused Celery workers to stop processing tasks after losing connection to their Redis broker.  This particular issue has been causing me a lot of trouble so I was wondering when to expect the next Kombu release?
Apologies if this isn't the right place to ask. 

Maurits

**Comment by Nusnus** [comment, marked as answer]
Hey there 👋

You’re definitely right, this fix requires a faster release cycle - it is a critical bug.

I’ve released an RC - please check if the issue is resolved and share your feedback (here is fine + tagging me): https://github.com/celery/kombu/releases/tag/v5.4.0rc1

Thank you!

**Reply by MauritsMonteyne** [reply]
@Nusnus Thanks for the quick reply. Finally had some time to test it and it looks like this RC fixed the issue for me!

**Reply by Nusnus** [reply]
Great news!! 🥳
Thank you for the feedback - it is very important and much appreciated!

**Comment by Nusnus** [comment]
[v5.4.0rc2](https://github.com/celery/kombu/releases/tag/v5.4.0rc2) is out!

v5.4.0 release will be in 1-2 weeks.
